### ISSUE TITLE
Countdown Timer Before Start

### ISSUE THREAD

**Issue by AdamD2** [xunkar/llanfair#5]
Other split programs have the option of counting down a specified number of seconds after pressing the start run button before the first split is activated. As an example it is commonly used to start the countdown timer at a game menu and start the run timer when the player gains control of the character. Is it possible/probable for this to be added? Thanks.

**Comment by xunkar** [comment]
Yes, definitely something I want to have in this overhaul. Meanwhile, I can suggest you look into Gered's fork of the original Llanfair, he already added support for such a thing https://github.com/gered/Llanfair
